editor/editor_node.cpp

```diff
@@ -4726,9 +4726,9 @@ void EditorNode::get_preload_scene_modification_table(
 				new_additive_node_entry.owner = p_node->get_owner();
 				new_additive_node_entry.index = p_node->get_index();
 
-				Node2D *node_2d = Object::cast_to<Node2D>(p_node);
-				if (node_2d) {
-					new_additive_node_entry.transform_2d = node_2d->get_transform();
+				CanvasItem *canvas_item = Object::cast_to<CanvasItem>(p_node);
+				if (canvas_item) {
+					new_additive_node_entry.transform_2d = canvas_item->get_transform();
```

Comment:
This preserves the transform data of CanvasItems, though the data still doesn't end up being used currently. I didn't want to change too much in this PR but this would enable us to support Control nodes in `EditorNode::reload_instances_with_path_in_edited_scenes()` later if we wanted to.